src/constants/themes.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+    themes: {
+
+        /* Official Themes */
+
+        "default": {
+            name: "Default Theme",
+            author: "SkyCrypt Team",
+            official: true,
+            logo: "https://sky.shiiyu.moe/resources/img/logo_square.png",
+            backgrounds: {
+                "bg": {
+                    "png": "https://sky.shiiyu.moe/resources/img/bg.png?v3",
+                    "webp": "https://sky.shiiyu.moe/resources/img/bg.webp?v3"
+                },
+                "bg_blur": {
+                    "png": "https://sky.shiiyu.moe/resources/img/bg_blur.png?v3",
+                    "webp": "https://sky.shiiyu.moe/resources/img/bg_blur.webp?v3"
+                }
+            },
+            colors: {
+                icon: "#0BCA51",
+                line: "#0BDA51",
+                link: "#0BEA51",
+                hover: "#09EF70",
+                maxed: "#DD980E",
+                gold: "#FDBB3C",
+                skillbar: "#0BAA51",
+                maxedbar: "#CE8F12"
+            }
+        },
+        "skylea": {
+            name: "sky.lea.moe's Theme",
```

Comment:
```suggestion
            name: "sky.lea.moe",
```
imo "sky.lea.moe" sounds better than "sky.lea.moe's theme"